releases/v21.2.2.md

```diff
@@ -0,0 +1,228 @@
+- Added support for a new [`index hint`](../v21.2/table-expressions.html#force-index-selection), `NO_FULL_SCAN`, which will prevent the optimizer from planning a full scan for the specified table. The hint can be used in the same way as other existing index hints. For example, `SELECT * FROM table_name@{NO_FULL_SCAN};`. Note that a full scan of a partial index may still be planned, unless `NO_FULL_SCAN` is forced in combination with a specific partial index via `FORCE_INDEX=index_name`. [#71437][#71437]
+- [`EXPLAIN ANALYZE (DEBUG)`](../v21.2/explain-analyze.html#debug-option) can now be used by [Serverless clusters](../cockroachcloud/quickstart.html). [#71969][#71969]
+- The [session variables](../v21.2/set-vars.html#supported-variables) `LC_COLLATE`, `LC_CTYPE`, `LC_MESSAGES`, `LC_MONETARY`, `LC_NUMERIC`, and `LC_TIME` were added for compatibility with PostgreSQL. They only support the `C.UTF-8` locale. [#72260][#72260]
+- Fixed an issue where the error message when creating the wrong type of forward index on a JSONB column was not clear. It now suggests creating an [inverted index](../v21.2/inverted-indexes.html). [#72361][#72361]
+- Changed the `pgerror` code from `CD` to `XC` for CockroachDB-specific errors. This is because the "C" class is reserved for the SQL standard. The `pgcode` `CDB00` used for unsatisfiable bounded staleness is now `XCUBS`. [#70448][#70448]
+- The [`CREATE TABLE ... LIKE ...`](../v21.2/create-table.html) statement now copies `ON UPDATE` definitions for `INCLUDING DEFAULTS`. [#70537][#70537]
+- CockroachDB now shows `indpred` on the [`pg_index` table](../v21.2/pg-catalog.html) for [partial indexes](../v21.2/partial-indexes.html). This was previously `NULL` for partial indexes. [#70884][#70884]
+- Fixed a bug where [`LINESTRINGZ`, `LINESTRINGZM`, and `LINESTRINGM`](../v21.2/linestring.html) could not be used as a column type. [#70747][#70747]
+- Added the [`crdb_internal.reset_index_usage_stats()` function](../v21.2/functions-and-operators.html) to clear index usage stats. This can be invoked from the SQL shell. [#72843][#72843]
+- The string "visible" is now usable as a table or column name without extra quoting. [#70563][#70563]
+- The `aggregation_interval` column has been added to the [crdb_internal.statement_statistics and crdb_internal.transaction_statistics](../v21.2/crdb-internal.html) tables, representing the aggregation duration of the respective statistics. [#72941][#72941]
+- The `diagnostics.sql_stats_reset.interval` [cluster setting](../v21.2/cluster-settings.html) was removed. In previous version of CockroachDB, in-memory SQL Statistics were reset periodically. This behavior is now removed since persisted SQL Statistics were introduced in v21.2. The `diagnostics.forced_sql_stats_reset.interval` cluster setting now only controls the reset of the reported SQL Statisticss if it is not collected by the telemetry reporter. [#72941][#72941]
```

Comment:
```suggestion
- The `diagnostics.sql_stats_reset.interval` [cluster setting](../v21.2/cluster-settings.html) was removed. In previous version of CockroachDB, in-memory SQL statistics were reset periodically. This behavior is now removed since persisted SQL Statistics were introduced in v21.2. The `diagnostics.forced_sql_stats_reset.interval` cluster setting now only controls the reset of the reported SQL statistics if it is not collected by the telemetry reporter. [#72941][#72941]
```